kalite/settings/base.py

```diff
@@ -314,8 +314,22 @@
 USE_L10N = getattr(local_settings, "USE_L10N", False)
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = getattr(local_settings, "SECRET_KEY",
-                     "8qq-!fa$92i=s1gjjitd&%s@4%ka9lj+=@n7a&fzjpwu%3kd#u")
+SECRET_KEY_FILE = os.path.join(USER_DATA_ROOT, "secretkey.txt")
+
+
+def generate_secret_key():
+    import uuid
+    import base64
+    key = base64.b64encode(str(uuid.uuid4()))
+    with open(SECRET_KEY_FILE, "w") as f:
+        f.write(key)
+        f.flush()
+
+if not os.path.exists(SECRET_KEY_FILE):
+    generate_secret_key()
```

Comment:
Rather than checking existence, I'd do a try-except around the full open-read-interpret block. That way, issues with format, etc will also trigger generating a new key.